github_issue_summarization/pachyderm_seldon_kvc/code/prediction.py

```diff
@@ -0,0 +1,31 @@
+import argparse
```

Comment:
Did these files originate in jupyter_argo_seldon/workflow/workspace/src? Everything inside jupyter_argo_seldon/workflow was added with the intention of being added to the walkthrough instructions, which hasn't happened yet. Since it's code that is useful for both approaches, I would like to store it in the base folder github_issue_summarization. If it's been modified slightly for this approach that's fine, but let's take it out of jupyter_argo_seldon/workflow instead of duplicating it.